db/migrate/20200211143240_split_up_user_preferences.rb

```diff
@@ -5,7 +5,7 @@ class SplitUpUserPreferences < ActiveRecord::Migration[5.2]
   def up
     User.find_ids_in_ranges(:batch_size => 20_000) do |min_id, max_id|
       DataFixup::SplitUpUserPreferences.
-        delay_if_production(priority: Delayed::LOW_PRIORITY, n_strand => ["user_preference_migration", Shard.current.database_server.id]).
+        delay_if_production(priority: Delayed::LOW_PRIORITY, n_strand: => ["user_preference_migration", Shard.current.database_server.id]).
```

Comment:
@StefSchenkelaars this fix is welcome - but this syntax is wrong. It should be 
```
  n_strand: ["user_preference_migration", Shard.current.database_server.id]
```

Reply:
Ow no, that's extremely stupid. I've just edited it in the GitHub editor because the change was so simple 🙈  Will fix